data/locale/German.json

```diff
@@ -46,42 +46,42 @@
     "Europe": "Europa",
 
     "Update available!": "Update verfügbar!",
-    "Press \uE000 to update to": "Drücke \uE000 um upzudaten nach:",
-    "Press \uE001 to cancel": "Drücke \uE001 um abzubrechen",
+    "Press \uE000 to update to": "Drücke \uE000 zum Aktualisieren nach:",
+    "Press \uE001 to cancel": "Drücke \uE001 zum Abbrechen",
     "Parsing JSON": "Parse JSON",
     "Updating, please wait...": "Update. Bitte warten...",
     "Internal server error!": "Interner Serverfehler",
-    "Invalid state value": "Invalide Serverantwort",
-    "Error creating directory": "Fehler beim erstellen des Ordners",
-    "Error extracting file": "Fehler beim entpacken",
-    "Error writing file": "Fehler beim schreiben",
+    "Invalid state value": "Ungültige Serverantwort",
+    "Error creating directory": "Fehler beim Erstellen des Ordners",
+    "Error extracting file": "Fehler beim Entpacken",
+    "Error writing file": "Fehler beim Schreiben",
     "Error opening file": "Kann Datei nicht öffnen",
-    "Error opening zip file": "Kann zip Datei nicht öffnen",
-    "Error extracting zip": "Fehler beim entpacken der zip",
-    "Error getting zip info": "Kann zip Infos nicht lesen",
-    "Error opening zip!": "Kann zip nicht öffnen",
-    "Aroma version too old to allow auto-updates": "Aroma Version zu alt für auto updates",
+    "Error opening zip file": "Kann ZIP Datei nicht öffnen",
+    "Error extracting zip": "Fehler beim Entpacken der ZIP",
+    "Error getting zip info": "Kann ZIP-Infos nicht lesen",
+    "Error opening zip!": "Kann ZIP nicht öffnen",
+    "Aroma version too old to allow auto-updates": "Aroma Version ist zu alt für automatische Updates",
     "Error creating temporary directory!": "Kann temporären Ordner nicht erstellen!",
-    "Error downloading": "Fehler beim Download von",
-    "Error getting own title info": "Konnte eigene Titel ID nicht abfragen",
-    "Error removing file": "Fehler beim löschen der Datei",
-    "Error removing directory": "Fehler beim löschen des Ordners",
-    "Error moving file": "Fehler beim verschieben der Datei",
-    "Error moving directory": "Fehler beim verschieben des Ordners",
+    "Error downloading": "Fehler beim Download von:",
+    "Error getting own title info": "Konnte eigene Titel Infos nicht abfragen",
+    "Error removing file": "Fehler beim Löschen der Datei",
+    "Error removing directory": "Fehler beim Löschen des Ordners",
+    "Error moving file": "Fehler beim Verschieben der Datei",
+    "Error moving directory": "Fehler beim Verschieben des Ordners",
 
     "Size:": "Größe:",
     "Provided title version": "Titelversion",
     "Only numbers": "Nur Nummern",
-    "LATEST": "LETZTE",
+    "LATEST": "LETZTER",
```

Comment:
This reffers to the version. So it should be `LETZTE Version`, not `LETZTER Version`